test/e2e/mcsd/update_test.go

```diff
@@ -124,6 +125,8 @@ func Test_mCSDUpdateClient_IncrementalUpdates(t *testing.T) {
 			},
 		}
 
+		time.Sleep(time.Second) // Server timestamps are in seconds
```

Comment:
We can probably get rid of all these `time.Sleep` calls if we use `RFC3339Nano` in our code (see PR suggestion earlier)